index.html

```diff
@@ -340,5 +340,27 @@ <h1>
         </div>
       </div>
     </section>
+    <div class="form-div">
+      <form action="" class="form">
+        <h2>Contact me</h2>
+
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <input type="text" maxlength="30" placeholder="Enter your name here" />
+        <input type="email" placeholder="Enter your email here" />
+        <textarea
+          name="text-area"
+          id="text-area"
+          cols="20"
+          rows="7"
+          maxlegnth="300"
+          placeholder="Write your message here"
```

Comment:
- [ ] Kindly make sure your `maxlength="500"` and you also spell the `maxlength` attribute correctly 🙏

Reply:
Fixed.